README.md

```diff
@@ -23,14 +23,34 @@ Include the SAML library.
 To use the saml library, we think in terms of service providers (e.g. Clever) and identity providers (e.g. partners that use ADFS).
```

Comment:
Can we make this more general? If I'm someone looking for a npm module to do SAML I don't know anything about Clever or our "partners". 

Also, as someone who works for Clever this description still makes no sense since our partners aren't IDPs...